src/components/NotificationBox.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { CSSProperties } from 'react';
+import { suomifiTheme, Icon } from 'suomifi-ui-components';
+import { Text } from 'components/ResponsiveComponents';
+import { NamespacesConsumer } from 'react-i18next';
+
+interface Props {
+  style?: CSSProperties;
+  notificationText?: string;
+}
+
+const NotificationBox = ({ style, notificationText }: Props): JSX.Element => (
+  <NamespacesConsumer>
+    {t => (
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          flexWrap: 'wrap',
+          padding: suomifiTheme.spacing.l,
+          margin: `${suomifiTheme.spacing.m} 0`,
+          ...style,
+          background: suomifiTheme.colors.accentSecondaryLight40,
+          fontSize: '24px',
```

Comment:
Do we have a typography token that could be used here?

Reply:
This is simply to set the size for the icon so it shouldn't make a big difference.